bin/cron/update_project_count

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+#
+#
+
+require 'sequel'
+require File.expand_path('../../../pegasus/src/env', __FILE__)
+require 'cdo/only_one'
+require 'cdo/properties'
+require src_dir 'database'
+require 'cdo/honeybadger'
+
+PEGASUS_DB = Sequel.connect(CDO.pegasus_db_writer.sub('mysql:', 'mysql2:'))
```

Comment:
Since this feature is inherently “reporting” functionality, it’s ok to query the reporting database, which is typically only out of date by a few milliseconds.  And that’s preferable, because we won’t negatively impact performance on the production database with searches across large datasets.  The Contact Rollups cron job `[has an example](url
https://github.com/code-dot-org/code-dot-org/blob/78bc09d7344fce667fc6f179b18468c0b5d36e8b/lib/cdo/contact_rollups.rb#L29)` of how to connect to the Reporting replica and how to override the 30 second max query execution timeout we have configured on both the production server and the reporting database.

Reply:
@sureshc Just pushed the changes to use the reporting database - can you check to make sure it looks right?

Reply:
Looks great!  See my comment about changing from iterating over the entire storage apps table with ruby logic to using Sequel `count` and `where` methods to construct a SQL query that does that logic in the database.